.azuredevops/pipelines/build-and-release.yaml

```diff
@@ -47,73 +47,51 @@ stages:
-
-          - task: UseNode@1
-            displayName: "Install Node.js from package.json version"
-            inputs:
-              version: $(PACKAGE_JSON_NODE_VERSION)
-
-          - task: DotNetCoreCLI@2
-            displayName: Restore dotnet tools
-            inputs:
-              command: custom
-              custom: tool
-              arguments: restore
-              workingDirectory: $(System.DefaultWorkingDirectory)
-
-          - task: DotNetCoreCLI@2
-            displayName: Restore dependencies
-            inputs:
-              command: restore
-              projects: ${{ variables.ProjectFilePath }}
-              feedsToUse: select
-              restoreArguments: --locked-mode
-
-          - task: DotNetCoreCLI@2
-            displayName: Build
-            inputs:
-              command: build
-              projects: ${{ variables.ProjectFilePath }}
-              configuration: ${{ variables.Configuration }}
-              arguments: --no-restore
-            env:
-              AuthenticodeClientSecret: $(AuthenticodeClientSecret)
-              # Roll-forward behavior set for AzureSignTool dotnet tool (see .config\dotnet-tools.json) which requires .Net 6.0 runtime
-              DOTNET_ROLL_FORWARD: Major
-
-          - task: DotNetCoreCLI@2
-            displayName: Create NuGet package
-            inputs:
-              command: pack
-              packagesToPack: ${{ variables.ProjectFilePath }}
-              configuration: ${{ variables.Configuration }}
-              packDirectory: $(System.DefaultWorkingDirectory)/packages
-              includesymbols: true
-              nobuild: true
-              versioningScheme: off
-
-          - publish: $(System.DefaultWorkingDirectory)/packages
-            displayName: Publish NuGet package as artifact
-            artifact: artifact
+        - task: DotNetCoreCLI@2
+          displayName: Restore dotnet tools
+          inputs:
+            command: custom
+            custom: tool
+            arguments: restore
+            workingDirectory: $(System.DefaultWorkingDirectory)
+
+        - task: DotNetCoreCLI@2
+          displayName: Restore dependencies
+          inputs:
+            command: restore
+            projects: ${{ variables.slnPath }}
+            feedsToUse: select
+            restoreArguments: --locked-mode
+
+        - task: DotNetCoreCLI@2
+          displayName: Build
+          inputs:
+            command: build
+            projects: ${{ variables.slnPath }}
+            configuration: ${{ variables.Configuration }}
+            arguments: --no-restore
+          env:
+            AuthenticodeClientSecret: $(AuthenticodeClientSecret)
+            # Roll-forward behavior set for AzureSignTool dotnet tool (see .config\dotnet-tools.json) which requires .Net 6.0 runtime
+            DOTNET_ROLL_FORWARD: Major
+
+        - template: steps.yml
```

Comment:
This template will not work unless you implement the steps.yml configuration. https://github.com/Kentico/xperience-by-kentico-crm/blob/main/.azuredevops/pipelines/steps.yml